app/node_modules/.package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "app",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Is this file and app/.package-lock.json from trying to run npm install in the app folder? I'm just wondering if they're necessary

Reply:
oh probably i didn't notice those because my github desktop is acting up and so I'm using the vscode git which I'm unfamiliar with